### PR TITLE
feat: add ObjectBox

### LINKED ISSUE #23: GreenDAO/ObjectBox
[GreenDAO](https://greenrobot.org/greendao/documentation/how-to-get-started/) is the fastest sqlite orm out there, so why not use it instead of Room?

There's also [ObjectBox](https://objectbox.io/dev-get-started/) from the same developers which claims to be even faster.

### PR DESCRIPTION
**Pull request by wvffle**
closes #23

### CODE REVIEW

**Comment by wvffle** [comment]
ObjectBox does not support String keys. We have to add https://github.com/wvffle/but-schedule-server/issues/1

File: .gitignore [hunk 1]
```diff
@@ -43,3 +43,6 @@ google-services.json
 
 # Android Profiling
 *.hprof
+
+# ObjectBox
```
Comment (wvffle) [inline .gitignore:47]: .

File: app/src/main/java/net/wvffle/android/pb/schedule/util/Serializer.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+public class Serializer {
+
+    private final static Serializer instance = new Serializer();
+    private Serializer() {}
+
+    public static Serializer getInstance() {
```
Comment (wvffle) [inline app/src/main/java/net/wvffle/android/pb/schedule/util/Serializer.java:21]: Mamy zaimplementowanego Singletona (#19)

File: app/src/main/java/net/wvffle/android/pb/schedule/ObjectBox.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import net.wvffle.android.pb.schedule.models.Teacher;
+import net.wvffle.android.pb.schedule.models.Title;
+import net.wvffle.android.pb.schedule.models.Update;
+
+import io.objectbox.Box;
+import io.objectbox.BoxStore;
+import io.objectbox.android.AndroidObjectBrowser;
+
+public class ObjectBox {
+    private static BoxStore boxStore;
```
Comment (wvffle) [inline app/src/main/java/net/wvffle/android/pb/schedule/ObjectBox.java:24]: A tu mamy fasade (#19)

**Review by Sylwia0506 — DISMISSED** [review]

**Review by Sylwia0506 — APPROVED** [review]

**Review by Sylwia0506 — APPROVED** [review]

**Pull request merged**